SU2_CFD/src/fluid/CSU2TCLib.cpp

```diff
@@ -52,10 +52,17 @@ CSU2TCLib::CSU2TCLib(const CConfig* config, unsigned short val_nDim, bool viscou
   eve_eq.resize(nSpecies,0.0);
   eve.resize(nSpecies,0.0);
 
-  if(viscous){
+  if (viscous) {
     MolarFracWBE.resize(nSpecies,0.0);
     phis.resize(nSpecies,0.0);
     mus.resize(nSpecies,0.0);
+
+    /*--- Reference values for Sutherland's law viscosity ---*/
+    //Note: Can be extended with better coefficients of individual species,.
+    k_ref.resize(1,0.0);
+    mu_ref.resize(1,0.0);
+    Sm_ref.resize(1,0.0);
+    Sk_ref.resize(1,0.0);
```

Comment:
Ideally, we would include values of each species in the gases.  However, for now, the general constants can be used.  Its hard finding values for monoatomic species.